test/integration/JsonSerialization.test.ts

```diff
@@ -147,4 +148,14 @@ describe("Testing the JSON serialization logic of a generated class", () => {
   }
 }`);
   });
+
+  it("should initialize Bundle's resource type by default", () => {
+    const resource: Bundle = new Bundle();
+    expect(resource.resourceType).toEqual("Bundle");
+  });
+
+  it("should initialize ResourceChild's resource type by default", () => {
```

Comment:
Codacy found an issue: [Unsafe call of an any typed value.](https://app.codacy.com/gh/MeasureAuthoringTool/model-info-parser/pullRequest?prid=6219961)